gt-elasticsearch/src/main/java/mil/nga/giat/data/elasticsearch/ElasticAttribute.java

```diff
@@ -198,8 +223,53 @@ public boolean equals(Object obj) {
             equal &= Objects.equals(analyzed, other.analyzed);
             equal &= Objects.equals(stored, other.stored);
             equal &= Objects.equals(nested, other.nested);
+            equal &= Objects.equals(order, other.order);
+            equal &= Objects.equals(customName, other.customName);
         }
         return equal;
     }
-
+    
+    /**
+     * Implement comparison logic
+     * @param o is a non-null ElasticAttribute
+     * @return negative for before, zero for same, positive after 
+     */
+    @Override
+    public int compareTo(ElasticAttribute o) {
+        if(this.order == null) { 
+            if(o.order == null)
+                return this.name.compareTo(o.name);
+            return 1;
+        }
+        if(o.order == null)
+            return -1;
+     
+        int i = this.order.compareTo(o.order);
```

Comment:
Our end users access a custom application that display fields in the order returned from GeoServer (unlike most applications which will order them alphabetically, no matter the order GeoServer returns the fields).  They asked for the ability to order these fields so they could place those with the highest importance on top.

Reply:
Okay, thanks. Can you update the [documentation](https://github.com/ngageoint/elasticgeo/blob/master/gs-web-elasticsearch/doc/index.rst#configuring-layer) to include the order and customName layer configuration items? As part of that you can update [elasticsearch_fieldlist.png](https://github.com/ngageoint/elasticgeo/blob/master/gs-web-elasticsearch/doc/images/elasticsearch_fieldlist.png). In case it's helpful, using your branch I generated an updated version of the image, which I think you can use to replace the previous version.

<img width="791" alt="elasticsearch_fieldlist" src="https://user-images.githubusercontent.com/11000076/50939574-13427e80-1443-11e9-81bf-0ffa2d6f22fa.png">

Reply:
OK, thanks for the image, I'll include that updated png.  I'll add the order and customName to the documentation, let me know if you have any comments on the description I used.